places to visit app/view models/SearchResultsVCViewModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SearchResultsVCViewModel.swift
+//  places to visit app
+//
+//  Created by Harriette Berndes on 21/07/2021.
+//
+
+import Foundation
+import MapKit
+
+class SearchResultsVCViewModel {
+    var delegate: SearchResultsViewControllerDelegate?
```

Comment:
I think this delegate should be `weak` otherwise you'll have a retain cycle (memory leak) between: `SearchResultsVCViewModel` <-> `SearchResultsViewController `